docs/configurations.md

```diff
@@ -8,30 +8,31 @@ nav_order: 7
 
 Configuration reference for `propeller.toml`.
 
-| Config                             | Values          | Description                                                                                                                                |
```

Comment:
what is the change in this file?

Reply:
Please add documentation for this config here

Reply:
I was adding the new option from `propeller.toml` to the docs. But since the docs were updated the day before yesterday and I didn't sync my fork, it caused a merge conflict. So I had to remove the changes to solve it.

Reply:
Now I have synced the fork, got the code up to date, and made the changes in the docs.